spec/models/refinery/jobs/job_application_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
+require 'support/helper/generator_helper'
+
+module Refinery
+  module Jobs
+    describe JobApplication, type: :model do
+      before :each do
+        @job_application = FactoryGirl.create :job_application
+        @job = FactoryGirl.create :job
+      end
+
+      context 'validations' do
+        it 'should be functional' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.